sklearn/discriminant_analysis.py

```diff
@@ -530,6 +529,11 @@ def fit(self, X, y):
 
         y : array-like of shape (n_samples,)
             Target values.
+
+        Returns
+        -------
+        self : returns an instance of self.
```

Comment:
```suggestion
        self :
```

Reply:
If there is no type, it should actually be,
```suggestion
        self
```
though this doesn't seem to be enforced by numpydoc https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
Actually for consistency at `fit`, we always put `self : object`.